### ISSUE TITLE
Excluding context in mappings breaks reindexing

### ISSUE THREAD

**Issue by simonpoole** [komoot/photon#411]
In mappings.json we exclude the context fields from the stored source document 	
	
```
"_source": {
    "excludes": [
        "context.*"
    ]
},
```
As the context fields are populated by the importer and updater, this implies that we can't reindex in situ (aka any change in the mappings or similar requires a reimport). I can see the space savings in doing this, but on the other hand it means that we can't evolve the mappings and related aspects of Photon in a "user"-friendly fashion.

**Comment by lonvia** [comment]
I would guess that the majority of users downloads the precomputed database provided by Graphhopper and rather repeat the download than reindexing the database. So I'd consider size a much more important argument.

**Comment by simonpoole** [comment]
That is true now given that the choice isn't actually available, but given a lightweight way to update, I suspect that will change essentially immediately.

**Comment by lonvia** [comment]
I still doubt that the average user will do that. If you want to provide a development mode with a reindexable database, now that would be a different story.

Please also note the discussion in #291.
